modules/documents/app/models/document.rb

```diff
@@ -29,8 +29,19 @@
 #++
 
 class Document < ApplicationRecord
+  enum :kind, {
+    legacy: "legacy",
+    collaborative: "collaborative"
+  }
```

Comment:
I might be missing some product context here, but is legacy what we were calling "document based"? Or are we misisng an entry here that will come in the future?

Reply:
So, as we don't have the file based one implemented, there's no need to add it

Reply:
genau... yet 🙂